actions/src/email_actions.py

```diff
@@ -17,6 +38,9 @@ def __init__(self, *args, config: Dict = None, **kwargs):
         self._floating_ip_api: OpenstackServer = config.get(
             "openstack_floating_ip_api", OpenstackFloatingIP()
         )
+        self._image_api: OpenstackImage = config.get(

```

Comment:
Longer-term I can see this splitting across action files, i.e. `openstack.image.email_users` since it's a code smell of this action gradually turning into a god action.

Next time we go to add an email action can we split this down. For the moment it's fine as it is though.